src/Core/Framework/Adapter/Asset/AssetInstallCommand.php

```diff
@@ -37,13 +41,21 @@ protected function configure(): void
         $this->addOption('force', 'f', InputOption::VALUE_NONE, 'Force the install of assets regardless of the manifest state');
     }
 
+    /**
+     * @throws \JsonException
+     * @throws UnableToDeleteDirectory
+     * @throws UnableToCreateDirectory
+     * @throws UnableToCheckExistence
+     * @throws FilesystemException
+     */
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $io = new ShopwareStyle($input, $output);
+        $io->title('Copying assets');
 
         foreach ($this->kernel->getBundles() as $bundle) {
             $io->writeln(\sprintf('Copying files for bundle: %s', $bundle->getName()));
-            $this->assetService->copyAssetsFromBundle($bundle->getName(), $input->getOption('force'));
+            $this->assetService->copyAssets($bundle, $input->getOption('force'));
```

Comment:
Just out of curiosity: Why is the other method now used?

Reply:
if we would use the method, that only takes the bundle name, we would need to search for the bundle instance again, but as we already have it available here, it is not necessary and we can use the better suiting method